pkg/codegen/docs/templates/examples.tmpl

```diff
@@ -0,0 +1,15 @@
+{{ define "examples" -}}
+{{ print "{{% examples %}}" }}
+## Example Usage
+
+{{- range . }}
+{{ .Title }}
```

Comment:
Are the titles going to be the same in each language? If so, do we need generate a unique id so they can be anchor linked?

Reply:
Great point, @justinvp ! Yes, the title, if present, will be the same across all languages for a specific example. So yes, having a unique id would be great. Thankfully, @sean1588's work to handle languages in anchor tags will make it all work, just as long as I ensure there is a unique ID for these titles.

Reply:
Actually, I just realized that the way this title is placed currently, it is _not_ inside the language snippet, so generating an ID per-language isn't needed. However, that also means someone sharing a link to the example wouldn't be able to pin to a specific language. Hmm curious what @cnunciato thinks as well.

Reply:
> I just realized that the way this title is placed currently, it is not inside the language snippet, so generating an ID per-language isn't needed.

Correct, and intentional, yeah. If we want to link to a specific language, we should be able to use the `?language=` querystring approach. (Which IMO we can do in a follow-up.)

Reply:
> Which IMO we can do in a follow-up.

Yeah that's what I am thinking too.